data/migrations/34.lua

```diff
@@ -38,30 +21,27 @@ function onUpdateDatabase()
 		result.free(resultId)
 	end
 
-	local resultId = db.storeQuery(string.format("SELECT `player_id`, `key`, `value` FROM `player_storage` WHERE `key` >= %d AND `key` <= %d", mountRange, mountRange + 10))
-	if resultId then
-		repeat
-			local playerId = result.getNumber(resultId, "player_id")
-			local key = result.getNumber(resultId, "key")
-			local mounts = {}
-
-			for i = 1, 300 do
-				if key == mountRange + (i % 31) then
-					table.insert(mounts, i-1)
-				end
-			end
-
-			for k, v in pairs(mounts) do
-				local lshift = bit.lshift(1, (v % 31))
+	-- this equals to our current id range from mounts
+	for i = 1, 200 do
+		local key = mountRange + ((i-1) / 31)
+		local resultId = db.storeQuery(string.format("SELECT `player_id`, `key`, `value` FROM `player_storage` WHERE `key` = %d", key))
```

Comment:
Maybe use `key >= 1 AND key <= 200` to avoid running 200 queries?

Reply:
Since you also fetch the `key`, you can use the value returned instead of `i` for the `INSERT`  below

Reply:
does this look better to you? https://github.com/otland/forgottenserver/pull/4197/commits/218cc896a9082d8ced0f1ce1724a8d03bb7b2644